modules/RoyalCommands/src/main/resources/deathmessages.yml

```diff
@@ -0,0 +1,151 @@
+###################
+# Variables!
+# {player} <- name of player who died
+# {dispplayer} <- display name of player who died (nicknames, etc)
+# {killer} <- what killed it (attacker name - PvP only)
+# {dispkiller} <- what killed it (attacker display name - PvP only)
+# {world} <- the world in which the player died
+# {mob} <- what killed it (if it's a monster)
+# {hand} <- what was in the killer's hand (PvP only, will use item name if available)
```

Comment:
This is the perfect time to align some of these with the existing variables with the larger RCmds. At a bare minimum, `kicks.messages.ingame_format` uses `{dispname}` rather than `{dispplayer}`. Though this might be difficult as most RCmds messages are single-player, whereas death has a player who died and a player who caused it.

Reply:
`dispname` makes sense for the killed player display name
Thoughts on using `kdispname` for the killer display name?